sdk/communication/azure-communication-sms/src/main/java/com/azure/communication/sms/SmsClientBuilder.java

```diff
@@ -217,15 +218,7 @@ private AzureCommunicationSMSServiceImpl createServiceImpl() {
         }
         HttpPipeline builderPipeline = this.pipeline;
         if (this.pipeline == null) {
-            HttpPipelinePolicy[] customPolicyArray = null;
-            if (customPolicies.size() > 0) {
```

Comment:
Why we don't need to add customPolicies anymore?

Reply:
Nevermind. I see that is done in line 257

Reply:
Thanks, Sam, I'll resolve then